src/client/assets/svgs/eye-crossed.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
could we use the icons provided by the design system? http://theguardian.design/2a1e5182b/p/3892a2-icons

Reply:
if all the icons are available through the design system, then we may not need all the svg importer changes

Reply:
👍  my first time seeing theguardian.design 😆  
looks very useful